File: 1. Two Sum/two_sum.c
#include <stdio.h>

void get_target (int* nums, int target, int* out ){
	int len_nums = sizeof(nums);
	//len_nums++;
	
	for (int i = 0; i <= len_nums; i++){
		int exit_loop = 0;
		for (int j = 0; j < len_nums; j++){
			if (i == j){
				continue;
			} else {
				if ((nums[i] + nums[j]) == target){
					out[0] = nums[i];
					out[1] = nums[j];
					exit_loop = 1;
				}
			}
		}
		if (exit_loop){
			break;
		}
	}
}

void main (){
	int number_list[] = {1,2,3,4,5,6,7,8,9};
	int target = 16;
	int indices[2];
	
	get_target(number_list, target, indices);
	
	printf("[%d,%d]\n\n", indices[0], indices[1]); 
}

File: 1. Two Sum/two_sum.js
function getTarget (nums, target){
	let lenNums = nums.length();
	let out = null;
	
	for (let a = 0; a < lenNums; a++){
		let exit_loop = false;
		for (let b = 0; b < lenNums; b++){
			if (b == a){
				continue;
			} else {
				if ((nums[a] + nums[b]) == target){
					out = [a,b];
					exit_loop = true;
					break;
				}
			}
		}
		
		if (exit_loop){
			break
		}
	}
	
	return out;
}

numberList = [1,2,3,4,5,6,7,8,9];
target = 16;

let indices = getTarget(numberList, target);
console.log (indices);

if (!(indices === null)){
	console.log (numberList[indices[0]]);
	console.log (numberList[indices[1]]);
}
